implementations/rust/ockam/ockam_abac/src/policies/storage.rs

```diff
@@ -0,0 +1,146 @@
+use crate::policies::ResourceTypePolicy;
+use crate::{
+    Action, Env, Expr, PolicyAccessControl, ResourceName, ResourcePoliciesRepository,
+    ResourcePolicy, ResourceType, ResourceTypePoliciesRepository,
+};
+use ockam_core::compat::sync::Arc;
+use ockam_core::compat::vec::Vec;
+use ockam_identity::{Identifier, IdentitiesAttributes};
+use tracing::{debug, instrument};
+
+#[derive(Clone)]
+pub struct Policies {
```

Comment:
Honestly, now I'm not sure why we need that struct

Reply:
I'm using it as a simplified "view" of both `ResourcePolicy` and `ResourceTypePolicy`. These two types can be converted to `Policy`. In that conversion, we compute the right policy (whatever it is, compounded or exclusive). It also helps to unify the output of the NodeManager functions used by the command.